examples/Autopoint/plot_hmc.py

```diff
@@ -0,0 +1,107 @@
+# VolEsti (volume computation and sampling library)
```

Comment:
Do we have the same file within the project?

Reply:
Yup. I will delete this one